spec/main_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require './lib/main'
+
+RSpec.describe Enumerable do
+  describe '#my_count' do
+    it 'calls a block on an object and returns the number of elements' do
+      arr = [1, 2, 3, 4, 5]
+      expect(arr.my_count).to eql(5)
+    end
+  end
+
+  describe '#my_map' do
+    it 'returns a new array as a result of calling a block once on each item in the original array' do
+      arr = [1, 2, 3]
+      result = [2, 4, 6]
+      expect(arr.my_map { |x| x * 2 }).to(eql(result))
+    end
+  end
+
+  describe '#my_any?' do
+    it 'returns true if the block passed returns true for any item in the array' do
+      arr = [1, 2, 3]
+      expect(arr.my_any? { |x| x > 2 }).to(eql(true))
```

Comment:
:thinking: What happens when the block returns false for each value? For example: :point_down: 
```suggestion
      expect(arr.my_any? { |x| x > 2 }).to(eql(true))
      expect(arr.my_any? { |x| x > 5 }).to(eql(false))
```
Please, let's add edge cases for the test. This test is not yet effective as it passes even like this (Commenting out part of the code). :point_down: 
```ruby
  def my_any?(&block)
    counter = 0
    my_each do |elem|
      counter += 1 if block.call(elem) == true
    end
    return true if counter.positive?

    # false
  end
```